applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -126,6 +126,28 @@ static void le_audio_event_publish(enum le_audio_evt_type event)
 	ERR_CHK(ret);
 }
 
+static int codec_freq_get_hz(const struct bt_audio_codec_cfg *codec)
```

Comment:
This code is duplicated. We can discuss another place to put it which is accessible for both the broadcast_sink and broadcast_source.

Reply:
Moved to le_audio.[c, h] as broadcast and unicast can access this, plus it is codec related.